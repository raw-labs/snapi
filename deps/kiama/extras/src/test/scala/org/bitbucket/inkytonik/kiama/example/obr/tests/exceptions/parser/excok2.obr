(* Legal RAISE statement *)

PROGRAM ExcOk2 (i : INTEGER) : INTEGER;

BEGIN

   RAISE CountException;

END ExcOk2.
