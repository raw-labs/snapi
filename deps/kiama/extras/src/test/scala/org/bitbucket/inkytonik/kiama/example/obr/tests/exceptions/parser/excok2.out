ObrInt(ExcOk2,Vector(IntParam(IdnDef(i))),Vector(RaiseStmt(IdnUse(CountException))),ExcOk2)
